core/src/main/java/tc/oc/pgm/wool/WoolMatchModule.java

```diff
@@ -205,19 +206,22 @@ public void placementCheck(final BlockTransformEvent event) {
 
   @EventHandler
   public void handleWoolCrafting(PrepareItemCraftEvent event) {
-    ItemStack result = event.getRecipe().getResult();
-    InventoryHolder holder = event.getInventory().getHolder();
-
-    if (holder instanceof Player) {
-      MatchPlayer playerHolder = this.match.getPlayer((Player) holder);
-
-      if (playerHolder != null && result != null && result.getType() == Material.WOOL) {
-        for (MonumentWool wool : this.wools.values()) {
-          if (wool.getDefinition().isObjectiveWool(result)) {
-            if (!wool.getDefinition().isCraftable()) {
-              playerHolder.sendWarning(
-                  translatable("wool.craftingDisabled", wool.getComponentName()));
-              event.getInventory().setResult(null);
+    Recipe recipe = event.getRecipe();
+    if (recipe != null) {
```

Comment:
```suggestion
    Recipe recipe = event.getRecipe();
    if (recipe == null) return;
```
Avoids the whole method being pushed one indentation level in

Reply:
Will do